### PR TITLE
Build UMD module with rollup

### PR DESCRIPTION
**Pull request by gustavnikolaj**
This is not strictly necessary, but it was an interesting exercise, as rolliup could potentially be the tool to use for sharing components between react applications.

I may take this a step further and rewrite some smaller parts of the code to ES6 and introduce a dependency (maybe [`invariant`](https://github.com/zertosh/invariant)) to further explore the tools capabilities.

### CODE REVIEW

**Comment by gustavnikolaj** [comment]
Inspiration drawn from:
- https://github.com/rollup/rollup-starter-lib/tree/babel
- https://github.com/reactjs/redux

**Comment by papandreou** [comment]
More inspiration here: https://github.com/unexpectedjs/unexpected/pull/409, https://github.com/One-com/one-color/pull/39

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import pkg from './package.json';
+import commonjs from 'rollup-plugin-commonjs';
+
+export default [
+  {
+    format: 'umd',
+    moduleName: 'Dislocator',
+    plugins: [commonjs()],
+    entry: 'lib/dislocator.js',
+    dest: pkg.browser
```
Comment (papandreou) [inline rollup.config.js]: No source maps? (you can just add `sourceMap: true` to the config here :)

**Review by papandreou — APPROVED** [review]

File: rollup.config.js (hunk 1 above)
Comment (gustavnikolaj) [reply rollup.config.js]: Right! Thanks :-)

**Pull request merged**